app.js

```diff
@@ -4,8 +4,36 @@
 //
 // const path = require("path");
 const AB = require("ab-utils");
+const child_process = require("child_process");
 
 var controller = AB.controller("file_processor");
 // controller.afterStartup((cb)=>{ return cb(/* err */) });
 // controller.beforeShutdown((cb)=>{ return cb(/* err */) });
 controller.init();
+
+if (process.env.CLAMAV_ENABLED == "true") {
+	child_process.execFile(
```

Comment:
minor issue:  let's make sure this get's run through eslint + prettier.  It looks like tabs are being used for indentation.

Reply:
You're right. Normally my editor uses spaces. Not sure why it switched to tabs here.